site/blog/2016/jul/1/sphinx-and-rtd-for-writers.rst

```diff
@@ -0,0 +1,552 @@
+It feels pretty magical once it's set up,
+and takes the thankless task of deploying documentation out of your day.
+
+Recommended Versioning System
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Read the Docs only works with version control.
+This means that however you version your software,
+your documentation follows suit.
+This is great for integrating with your development team,
+but it also means you need to think about the proper strategy for versioning.
+
+If you work on a SaaS product,
+you probably only have one version.
+However,
+if you produce software with multiple supported versions,
+we recommend the following:
+
+* A ``master`` branch that the next release of your software is developed on
+* Git *branches* for ongoing maintenance of each version of your software that is maintained
+* Git *tags* for specific released versions that users might be using
+
+We recommend release branches because it allows you to update them over time.
+Git *tags* are static,
+so they are appropriate for specific versions that a user might have installed.
+
+An example:
+
+* ``master`` is your 2.2 release that isn't out yet
+* ``2.1.X`` is your release branch for the ``2.1`` version
+* ``2.1.0`` is a tag of your ``2.1`` branch, when you release ``2.1.0`` to your users
+* ``2.1.1`` is a similar tag of your ``2.1`` branch
+* ``1.4.X`` is the latest release branch of your ``1.X`` version
+* ``1.4.3`` is the latest release of your ``1.X`` version
+
+Additional Read the Docs features
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Read the Docs also provides the following features:
+
+* GitHub, BitBucket, and Gitlab post-commit hooks
+* Custom domain hosting (CNAMEs)
+* Full-text search for all your projects versions
+* Status badges to show your docs are up to date
+* Hosting of multiple languages for a specific project
+* Hosting of multiple projects on a single domain with "subprojects"
+
+Feel free to email me or find me at a conference if you want to talk more about these things,
+or have ideas for other neat features.
+
+Real Life Examples
+------------------
+
+Read the Docs has a large number of users across many different programming languages.
+This is in part because Sphinx is such a powerful and dynamic tool,
+and Read the Docs makes it easy and free to host docs for your open source project.
+
+Here are some examples that show the wide range of projects using Sphinx & Read the Docs:
+
+* Julia - A language for scientific computing: http://docs.julialang.org
+* Asp.Net - Microsoft's web framework: https://docs.asp.net
+* Jupyter - An interactive programming environment: http://jupyter.readthedocs.io
+* PHPMyAdmin - A web-based database editor: https://docs.phpmyadmin.net
+* Write the Docs - The community site for Write the Docs - http://www.writethedocs.org
+
+As you can see,
+a wide range of projects are using the tools for many different uses.
+It's a powerful tool for writing prose as well as documenting source code.
+
+Conclusion
```

Comment:
I'm personally rubbed the wrong way by explicit section titles that describe the following paragraph's purpose. This might just be me though. I find there is always a better way to say "Introduction" or "Conclusion", or to segue fluidly without the need for a call out.